### PR TITLE
Add cfx.re as hard

### PR DESCRIPTION
**Pull request by Rathmox**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -3619,6 +3619,18 @@
         ]
     },
 
+    {
+        "name": "CEX.IO",
```
Comment (tupaschoal) [inline _data/sites.json:3623]: Shouldn't it be `cfx.re`?

Comment (Rathmox) [reply _data/sites.json:3623]: Yes indeed, I mistyped

Comment (Rathmox) [reply _data/sites.json]: No, I did not mistype, just forgot to change the title

File: _data/sites.json [hunk 2]
```diff
@@ -3619,6 +3619,18 @@
         ]
     },
 
+    {
+        "name": "Cfx.re",
+        "url": "https://support.cfx.re/hc/en-us/requests/new",
+        "difficulty": "hard",
+        "notes": "You need to send a support request. The button in your forum profile page leds you to that.",
```
Comment (tupaschoal) [inline _data/sites.json:3626]:
```suggestion
        "notes": "You need to send a support request. The button in your forum profile page leads you to that.",
```

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**